app/components/LogIn/Login.test.js

```diff
@@ -0,0 +1,49 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import LogIn from './LogIn';
+
+
+
+describe('LogIn Component', () => {
+const mockUser ={
+  user: {
+    id: 1,
+    name: 'Taylor',
+    password:'password',
+    email: 'tman2272@aol.com'
+  }
+}
+
+const wrapper = shallow(
+  <LogIn signIn={jest.fn()} user={mockUser}/>
+)
+
+  it('displays error', async(done) => {
+
+    fetchMock.post('https://api.themoviedb.org/3/movie/now_playing?api_key=e918cc56cafd311d7955d426f4da1685&language=en-US&page=1', { status: 500, body: {} })
```

Comment:
Why are you trying to match a POST to the movie API? Likely this is not an endpoint you're posting to. I think this URL should match the login API endpoint.